.github/CONTRIBUTING.md

```diff
@@ -6,4 +6,4 @@ In order to accept your pull request, we need you to [submit a CLA](https://gith
 
 ## License
 
-By contributing to Trino, you agree that your contributions will be licensed under the [Apache License Version 2.0 (APLv2)](LICENSE).
+By contributing to Trino, you agree that your contributions will be licensed under the [Apache License Version 2.0 (APLv2)](https://www.apache.org/licenses/LICENSE-2.0).
```

Comment:
```suggestion
By contributing to Trino, you agree that your contributions will be licensed under the [Apache License Version 2.0 (APLv2)](../LICENSE).
```

The link should point to the file in the root of the repo.

Reply:
Thanks, Updated.